lib/documents/schemas/licence_transactions.json

```diff
@@ -10,9 +10,6 @@
     "format": "licence_transaction"
   },
   "show_summaries": true,
-  "organisations": [
```

Comment:
I think we'll still want the organisations attribute as this also defines the finders organisation (from: ... on /find-licences). However, this is probably a good opportunity to swap the organisation from GDS -> BEIS as we only set this to GDS as we were drafting the prototype.

Reply:
Cool (but same question as below - BEIS no longer a thing, so is it DBT now?)

Reply:
Ah yes it is now DBT, doh!